src/Parallel/ConstGlobalCache.hpp

```diff
@@ -342,12 +344,14 @@ namespace PUP {
 template <typename Metavariables>
 inline void pup(PUP::er& p,  // NOLINT
                 Parallel::ConstGlobalCache<Metavariables>*& t) noexcept {
+  Parallel::CProxy_ConstGlobalCache<Metavariables>
+      local_const_global_cache_proxy;
   if (p.isUnpacking()) {
-    Parallel::CProxy_ConstGlobalCache<Metavariables> const_global_cache_proxy;
-    const_global_cache_proxy.pup(p);
-    t = const_global_cache_proxy.ckLocalBranch();
+    p | local_const_global_cache_proxy;
+    t = local_const_global_cache_proxy.ckLocalBranch();
   } else {
-    t->get_self_proxy().pup(p);
+    local_const_global_cache_proxy = t->get_self_proxy();
+    p | local_const_global_cache_proxy;
```

Comment:
[not a change request] Help: how are the two new lines different from the old line?

Should the changes to this file be moved into the earlier commit for const global cache?